packages/runtime-dom/src/components/TransitionGroup.ts

```diff
@@ -170,8 +173,21 @@ function callPendingCbs(c: VNode) {
   }
 }
 
+function getRelativeBoundingClientRect(el: Element) {
+  const elRect = el.getBoundingClientRect()
+  if (!el.parentElement) return elRect
+  const parentRect = el.parentElement.getBoundingClientRect()
+
+  return new DOMRectReadOnly(
+    elRect.x - parentRect.x,
+    elRect.y - parentRect.y,
+    elRect.width - parentRect.width,
+    elRect.height - parentRect.height
```

Comment:
Is calculating the relative `width`/`height` actually necessary? It seems that these will typically end up as negative values.

Using `DOMRect` was maybe convenient when using `getBoundingClientRect` directly, but don't we just need `{ top: number, left: number }` in practice?

Reply:
You are right, it make no sense. Probably supposed to be `right` and `bottom`, but anyway, I got rid of them as unused variables.